### ISSUE TITLE
False positives for Option HexFormat / Strings in Quotes

### ISSUE THREAD

**Issue by webgefrickel** [brigade/scss-lint#52]
First things first: awesome tool! I just started using it in an existing project to highlight some code-smell and it helps a lot.

One small thing i found: it chokes on @each-statements with color-variables if they are not wrapped in quotes, e.g.

```
@each $item in
    item-blue,
    item-red {
    ...
}
```

I get:

items.scss:3 [W] Color `blue` should be written in hexadecimal form as `#00f`
items.scss:4 [W] Color `red` should be written in hexadecimal form as `#f00`

even though these strings are not even meant as color-declarations.

Wrapping those in quotes fixes those warnings:

```
@each $item in
    "item-blue",
    "item-red" {
    ...
}
```

After all, maybe it would be wise to always wrap strings in quotes in sass/compass (maybe this would make another nice option!) - nevertheless, wanted to report this :-)
